packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts

```diff
@@ -15,7 +15,7 @@ export function withSlatePasting<T extends Editor>(editor: T) {
             const fragment = decodeSlateFragment(slateFragment);
 
             if (isValidFragment(fragment)) {
-                Transforms.insertNodes(editor, fragment);
+                Transforms.insertFragment(editor, fragment);
```

Comment:
Originally was changed here https://github.com/prezly/slate/pull/210/files#diff-7dbb892cca06713d9888d229abe8a4ca0dcff541fe479aed3efc1cdbc117d847L26

was it intended?

Reply:
I believe I've changed it to fix the failing tests. Now they fail again :)

Reply:
yes, but maybe tests are wrong, `insertFragment` has been there for a long time i suppose

Reply:
Maybe the tests are wrong :man_shrugging:

Reply:
@e1himself fixed tests